common/app/views/fragments/img.scala.html

```diff
@@ -45,7 +45,9 @@
         <a href="@url#img-1" class="article__img-container js-gallerythumbs" data-link-name="Launch Article Lightbox" data-is-ajax>
             <div class="@if(isFeature && isShowcase){u-responsive-ratio u-responsive-ratio--letterbox}">
                 @image(picture, widths)
-                <span class="rounded-icon article__fullscreen"><i class="i i-expand-white"></i><i class="i i-expand-black"></i></span>
+                <span class="rounded-icon article__fullscreen">
```

Comment:
Changed the styling to suit this! Certainly cleaner https://github.com/guardian/frontend/blob/878c0f620b5bc442f17cac5516b73609b4d02a83/common/app/views/fragments/img.scala.html#L48.